City-of-Pori/pori-kada
# PORI-85 && 176: Landing-page and menu things for visitpori.

Landing-page and menu things for visitpori. Also added footer information block for visitpori.

To test:
1. drush updb -y && drush fra -y && drush cc all
2. Go to edit menus, add links to visitpori_main_menu_fi, in menu-item-attributes add class "visitpori-theme-color-red" (colors: red, orange, green, greenDark, blue), add few links to top-menu and footer-menu also
3. Confirm that menus are working.
4. Go to taxonomy and add a new target group for example "Tapahtumat"
5. Go to add content and create few liftups with Tapahtumat target group and a project flag, also create liftups with the carousel flag and same target group to lift something to the carousel.
6. Go to add content and create a new landing-page called "Tapahtumat" and set the target group as tapahtumat, also add some topic liftups and check few "Ajankohtais flow" things.
7. Confirm that the page lists the liftups that you created.
